enhancements/monitoring/monitoring-config.md

```diff
@@ -0,0 +1,307 @@
+---
+title: monitoring-config
+authors:
+  - "@marioferh"
+  - "@danielmellado"
+  - "@jan--f"
+  - "@moadz"
+  - "@simonpasquier"
+reviewers:
+  - "@jan--f"
+  - "@moadz"
+  - "@simonpasquier"
+approvers:
+  - "@jan--f"
+  - "@moadz"
+  - "@simonpasquier"
+  - "@openshift/openshift-team-monitoring"
+api-approvers:
+  - "@deads2k"
+  - "@JoelSpeed"
+creation-date: 2024-04-26
+last-updated: 20204-26
+tracking-link:
+- "https://issues.redhat.com/browse/MON-1100"
+  
+---
+
+# CRD ClusterMonitoring
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+* Currently, the OCP monitoring stacks are configured using ConfigMaps (1 for platform monitoring and 1 for user-defined monitoring). In OpenShift though the best practice is to configure operators using custom resources.
+
+
+## Motivation
+
+* The specification is well known and to a degree self-documenting.
```

Comment:
With proper godoc writing, "to a degree" could be dropped

Reply:
done